### PR TITLE
Update MCF Browser to use new DC API

### PR DESCRIPTION
**Pull request by cwcrystal8**
Repo has the following changes:

1. Change the following three API calls to use v1 DC REST API
  - Check if a DCID exists in the DC KG
  - Get all property labels for a DCID
  - Get a property value for a given DCID and property label

2. Rebuild static files

### CODE REVIEW

File: mcf-browser/src/back-end/utils.js [hunk 1]
```diff
@@ -61,13 +70,12 @@ async function getRemotePropertyLabels(dcid) {
 async function getRemotePropertyValues(dcid, label, isInverse) {
   const direction = isInverse ? 'in' : 'out';
   const targetUrl =
-      (API_ROOT + '/node/property-values?limit=500&dcids=' + dcid +
-       '&property=' + label + '&direction=' + direction);
+      (API_ROOT + '/v1/property/' + direction + '/' + label +
```
Comment (juliawu) [inline mcf-browser/src/back-end/utils.js:73]: We're updating the v1 REST API signature to /v1/property/values/{direction}/{dcid}/{property}
@shifucun to confirm: We should update to the new signature now, right?

File: mcf-browser/src/back-end/utils.js [hunk 2]
```diff
@@ -103,10 +111,13 @@ function getValueFromValueObj(valueObj) {
  *     Data Commons Knowledge Graph.
  */
 async function doesExistsInKG(dcid) {
-  const url = API_ROOT + '/node/triples?dcids=' + dcid + '&limit=1';
+  const url = API_ROOT + '/v1/property/out/typeOf/values/' + dcid;
```
Comment (juliawu) [inline mcf-browser/src/back-end/utils.js:114]: Same comment here: update to /v1/property/values/out/ + dcid + /typeOf

File: mcf-browser/src/back-end/utils.js (hunk 1 above)
Comment (cwcrystal8) [reply mcf-browser/src/back-end/utils.js:73]: Do you know when the new signature update is happening? Last time I spoke to @shifucun, I think he said to keep using the old signature for now.

File: mcf-browser/src/back-end/utils.js [hunk 3]
```diff
@@ -42,10 +42,19 @@ const ERROR_MESSAGES =
...
+  const outPropertyLabels = await fetch(outTargetUrl)
+      .then((res) => res.json())
+      .then((data) => data.properties);
+
+  // Get inward property labels
+  const inTargetUrl = API_ROOT + '/v1/properties/in/' + dcid;
+  const inPropertyLabels = await fetch(inTargetUrl)
       .then((res) => res.json())
-      .then((data) => JSON.parse(data.payload)[dcid]);
+      .then((data) => data.properties);
```
Comment (shifucun) [inline mcf-browser/src/back-end/utils.js:55]: You can fetch in and out properties concurrently like https://stackoverflow.com/a/59664248

File: mcf-browser/src/back-end/utils.js (hunk 1 above)
Comment (shifucun) [reply mcf-browser/src/back-end/utils.js:73]: Just updated the v1 API, please use the one Julia suggested.

Comment (cwcrystal8) [reply mcf-browser/src/back-end/utils.js:73]: Spoke to Bo and the new signature is live now, the next commit will have this change!

File: mcf-browser/src/back-end/utils.js (hunk 2 above)
Comment (cwcrystal8) [reply mcf-browser/src/back-end/utils.js:114]: Added in next commit!

File: mcf-browser/src/back-end/utils.js [hunk 4]
```diff
@@ -70,8 +70,8 @@ async function getRemotePropertyLabels(dcid) {
 async function getRemotePropertyValues(dcid, label, isInverse) {
   const direction = isInverse ? 'in' : 'out';
   const targetUrl =
-      (API_ROOT + '/v1/property/' + direction + '/' + label +
-      '/values/' + dcid);
+      (API_ROOT + '/v1/property/values/' + direction + '/' + dcid +
```
Comment (shifucun) [inline mcf-browser/src/back-end/utils.js:73]: nit: can use template literals for better readability: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

File: mcf-browser/src/back-end/utils.js (hunk 3 above)
Comment (cwcrystal8) [reply mcf-browser/src/back-end/utils.js]: Changed in next commit!

File: mcf-browser/src/back-end/utils.js (hunk 4 above)
Comment (cwcrystal8) [reply mcf-browser/src/back-end/utils.js]: Changed in next commit!

**Comment by cwcrystal8** [comment]
Changes since last review:
- API calls changed to new signature
- Sequential API calls switched to simultaneous API calls
- URLs formed with template literals instead of string addition

**Review by shifucun — APPROVED** [review]
thanks for the update

**Pull request merged**